response_operations_ui/controllers/mock_party_controller.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from flask import current_app as app
+
+
+def mock_search_respondents(first_name, last_name, email_address, page=1):
+    with open('response_operations_ui/data/mock_search_respondents_data.json', 'r') as json_data:
+        data = json.load(json_data)
+        limit = app.config['PARTY_RESPONDENTS_PER_PAGE']
+
+        chunk_range = range((len(data) + limit - 1) // limit)
+        data_by_surname = sorted(data, key=lambda dict: dict["lastName"].lower())
+        sorted_data = filter_json_by_passed_parameters(data_by_surname, first_name, last_name, email_address)
+        data_chunks = [sorted_data[i * limit:(i + 1) * limit] for i in chunk_range]
```

Comment:
maybe add a short doc string just to say what this does, obvious enough if you know imo

Reply:
Done. Mark resolved if you're happy